app/assets/javascripts/canvas_shim/application.js.erb

```diff
@@ -9,7 +9,8 @@
 //
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
-//= require jquery
+//
+//= require jquery/dist/jquery
```

Comment:
i would double check this file against master, i don't think we expect this file to have changed.

Reply:
yeah... looks like it got committed accidentally during a WIP commit. I'll fix that

Reply:
@adiaz04 - while I'm at it, should I changed this file back to `.js` instead of `.js.erb` since the shim_patch fix that was using `erb` is no longer relevant or should I leave it as is?

Reply:
it makes senses to remove the `.erb` since we're no longer doing that ruby logic that didn't work. :)